VictoriaAlbert/VictoriaAlbert/Object.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  Object.swift
+//  VictoriaAlbert
+//
+//  Created by Ana Ma on 11/10/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+enum ObjectModelParseError : Error {
+    case dictionary
+    case records
+    case fields
+    case primary_image_id
+    case rights
+    case year_start
+    case object_number
+    case artist
+    case museum_number
+    case object
+    case longitude
+    case last_processed
+    case event_text
+    case place
+    case location
+    case last_checked
+    case museum_number_token
+    case latitude
+    case title
+    case date_text
+    case slug
+    case sys_update
+    case collection_code
+}
+
+
+class Object {
+    let primary_image_id : String
+    let rights : Int
+    let year_start: Int
+    let object_number: String
+    let artist: String
+    let museum_number: String
+    let object : String
+    //    let longitude: String
+    let last_processed: String
+    let event_text: String
+    let place: String
+    let location: String
+    let last_checked : String
+    let museum_number_token : String
+    //    let latitude : String
+    let title : String
+    let date_text : String
+    let slug : String
+    let sys_updated: String
+    let collection_code: String
+    var imageURLString: String? { //Computed Property
```

Comment:
I really like this as a computed property, but you needed two: one for the larger image and one for the thumbnail. In your project by constructing the _jpg_o.jpg version every time you ended up with lo-res images on the detail page. Also, why are you checking for count >= 6. Were there exceptions to this or just bounds checking?